.travis.yml

```diff
@@ -6,12 +6,54 @@ sudo: required
 services:
   - docker
 env:
-install: 
+  global:
+    - PA_TEST_ONLINE_INSTALLER=true
+    - PYTHONPATH=$(pwd)
+  matrix:
+    - OTHER_TESTS=true
+    - PRODUCT_TEST_GROUP_SA_BARE="tests/product/test_installer.py tests/product/standalone/test_installation.py"
+    - PRODUCT_TEST_GROUP_SA_PRESTO="tests/product/test_authentication.py tests/product/test_configuration.py tests/product/test_server_upgrade.py"
+    - PRODUCT_TEST_GROUP_SA_PA="tests/product/test_error_handling.py tests/product/test_package_install.py tests/product/test_plugin.py tests/product/test_script.py tests/product/test_topology.py"
+    - PRODUCT_TEST_GROUP_SA_PA="tests/product/test_server_install.py"
+    - PRODUCT_TEST_GROUP_SA_PA_PRESTO="tests/product/test_status.py"
+    - PRODUCT_TEST_GROUP_SA_PA_PRESTO="tests/product/test_collect.py"
+    - PRODUCT_TEST_GROUP_SA_PA_PRESTO="tests/product/test_connectors.py"
+    - PRODUCT_TEST_GROUP_SA_PA_PRESTO="tests/product/test_control.py"
+    - PRODUCT_TEST_GROUP_SA_PA_PRESTO="tests/product/test_server_uninstall.py"
+    - PRODUCT_TEST_GROUP_YS_PA="tests/product/yarn_slider/test_slider_installation.py"
+    #-QUARANTINED="tests/product/yarn_slider/test_server_install.py"
+install:
   - pip install --upgrade pip==6.1.1
   - pip install -r requirements.txt
 before_script:
   - make docker-images
-script: 
-  - make clean lint dist docs 
-  - nosetests --with-timer --timer-ok 60s --timer-warning 300s -s tests.unit
-  - nosetests --with-timer --timer-ok 60s --timer-warning 300s -s tests.integration
+script:
+  - |
+    if [ -v PRODUCT_TEST_GROUP_SA_BARE ]; then
+      export IMAGE_NAMES="standalone_bare"
+      make test-images
+      tox -e py26 -- -a '!quarantine,!offline_installer' ${PRODUCT_TEST_GROUP_SA_BARE}
+    elif [ -v PRODUCT_TEST_GROUP_SA_PRESTO ]; then
+      export IMAGE_NAMES="standalone_presto"
+      make test-images
+      tox -e py26 -- -a '!quarantine,!offline_installer' ${PRODUCT_TEST_GROUP_SA_PRESTO}
+    elif [ -v PRODUCT_TEST_GROUP_SA_PA ]; then
+      export IMAGE_NAMES="standalone_pa"
+      make test-images
+      tox -e py26 -- -a '!quarantine,!offline_installer' ${PRODUCT_TEST_GROUP_SA_PA}
+    elif [ -v PRODUCT_TEST_GROUP_SA_PA_PRESTO ]; then
+      export IMAGE_NAMES="standalone_presto standalone_pa"
+      make test-images
+      tox -e py26 -- -a '!quarantine,!offline_installer' ${PRODUCT_TEST_GROUP_SA_PA_PRESTO}
+    elif [ -v PRODUCT_TEST_GROUP_YS_PA ]; then
+      export IMAGE_NAMES="yarn_slider_pa"
+      make test-images
+      tox -e py26 -- -a '!quarantine,!offline_installer' ${PRODUCT_TEST_GROUP_YS_PA}
+    elif [ -v OTHER_TESTS ]; then
+      make clean lint dist docs test-rpm
+      nosetests --with-timer --timer-ok 60s --timer-warning 300s -s tests.unit
+      nosetests --with-timer --timer-ok 60s --timer-warning 300s -s tests.integration
```

Comment:
I think there is no need to set timers for unit and integration tests, they are really short.